src/framework/filterControl/filterDropdown/_filterFormDropdown.scss

```diff
@@ -0,0 +1,43 @@
+
+.filterFormDropdown {
+  position: relative;
+}
+
+  .filterFormDropdown__form {
+    background: white;
+    border-radius: $addedFilter_borderRadius;
+    border: 1px solid $addedFilter_borderColor;
+    color: #474b4e;
```

Comment:
If this colour is available in colour values under `global/_colors.scss` then that value should be used.